examples/viewer.html

```diff
@@ -387,6 +423,23 @@
     }
 
 
+    function parseUrl(urlString) {
+      var a = document.createElement('a');
```

Comment:
think we can move this out of the function to just do 1 element creation. (we did that in url.js), granted this is only demo code so feel free to ignore.